src/Symfony/Component/Console/Command/Command.php

```diff
@@ -164,6 +164,9 @@ public function isEnabled(): bool
      */
     protected function configure()
     {
+        if (!$this->code && method_exists($this, '__invoke')) {
+            $this->code = new InvokableCommand($this, $this->__invoke(...));
+        }
```

Comment:
The only advantage of using the magic `__invoke` method is that you don't need to name it.

```suggestion
        if (!$this->code && is_callable($this)) {
            $this->code = new InvokableCommand($this, $this(...));
        }
```